README.md

```diff
@@ -12,7 +12,7 @@ QIWIS (**Q**u**I**qcl **W**idget **I**ntegration **S**oftware) is a framework fo
 ## How to download and install
 One of the below:
 - Using released version (_recommended_)
-1. `pip install git+https://github.com/snu-quiqcl/qiwis.git@v2.0.0`
+1. `pip install -e git+https://github.com/snu-quiqcl/qiwis.git@v2.0.1#egg=qiwis`
```

Comment:
Let's add more information about this.
How about adding the link: https://pip.pypa.io/en/stable/topics/vcs-support/

Reply:
Okay! I will add the link.